# Goal: which position to reach
geometry_msgs/Point target_position
---
# Result: final position
geometry_msgs/Pose2D final_position
---
# Feedback: current position
geometry_msgs/Pose2D base_position
